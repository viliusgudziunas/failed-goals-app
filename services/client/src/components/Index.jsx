import React from "react";

const Index = () => {
  return (
    <div>
      <h1>Goals</h1>
    </div>
  );
};

export default Index;
