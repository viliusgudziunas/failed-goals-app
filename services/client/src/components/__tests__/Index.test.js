import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import Index from "../Index";

describe("Index component", () => {
  it("should match the snapshot", () => {
    const tree = renderer.create(<Index />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  const component = shallow(<Index />);
  it("should have a header", () => {
    const header = component.find("h1");
    expect(header.length).toBe(1);
    expect(header.get(0).props.children).toBe("Goals");
  });
});
