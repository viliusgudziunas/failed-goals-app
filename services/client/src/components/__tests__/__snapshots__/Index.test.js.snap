// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index component should match the snapshot 1`] = `
<div>
  <h1>
    Goals
  </h1>
</div>
`;
